extensions/db-diagram/src/controllers/mainController.ts

```diff
@@ -3,7 +3,9 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// import * as azdata from 'azdata';
+
+import * as path from 'path';
+import * as fs from 'fs';
```

Comment:
import { promises as fs} form 'fs';

That way you're just using the promise based api